bin/test-example

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
```

Comment:
I usually favor `#!/usr/bin/env bash` as a rule, so that it uses the bash it finds on the PATH. I don't know if I tested that in Travis.

Using `#!/bin/bash` will break the script in NixOS - which is not a problem for most of the tracks - but Nix is specially popular among Haskell programmers.